kubernetes/op-scim-deployment.yaml

```
@@ -23,7 +23,7 @@ spec:
             - name: http
               containerPort: 3002
           volumeMounts:
-            - name: scimsession
+            - name: secrets
```

```suggestion
            - name: secret
```

Given the value for this is an arbitrary label, perhaps we should consider changing this to match the directory name.